### PR TITLE
set header height for IE.

### LINKED ISSUE #2492: Improve IE support.
If you use Steemit.com on Internet Explorer. The header is too big -  and makes some important user interactions impossible / not clear...

## e.g. 
- cannot close welcome banner
- once welcome banner is closed cannot select sort order etc..
- Sign up is painful...
- Posting is impossible.
screenshot from 2018-02-09 11-46-37
iain_004

### PR DESCRIPTION
**Pull request by gl2748**
closes #2492 
# Before - Impossible to write a post on IE.
image 1

# After:
image

### CODE REVIEW

**Comment by bnchdrff** [comment]
change in logo size & alighnemt with "beta":
before:
screenshot from 2018-02-09 15-11-49
after:
screenshot from 2018-02-09 15-11-36

**Comment by gl2748** [comment]
@bnchdrff fixed that beta alignment.

**Comment by gl2748** [comment]
# before:
screenshot from 2018-02-09 21-59-03

# after:
screenshot from 2018-02-09 21-59-13

File: src/app/components/modules/Header.jsx [hunk 1]
```diff
@@ -262,8 +262,6 @@ class Header extends React.Component {
                                         </svg>
                                         <svg
                                             className="logo-new logo-new--desktop"
-                                            width="148px"
-                                            height="38px"
                                             viewBox="0 0 148 38"
```
Comment (bnchdrff) [inline src/app/components/modules/Header.jsx]: should we change the viewBox to correspond with the new dimension (144px)?

File: src/app/components/modules/Header.jsx [hunk 2]
```diff
@@ -262,7 +262,7 @@ class Header extends React.Component {
                                         </svg>
                                         <svg
                                             className="logo-new logo-new--desktop"
-                                            width="148px"
+                                            width="144px"
                                             height="38px"
                                             viewBox="0 0 148 38"
```
Comment (bnchdrff) [inline src/app/components/modules/Header.jsx]: still 148px here!

**Pull request merged**